portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,26 +50,39 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     ArrayList<String> comments = new ArrayList<String>();
+    ArrayList<String> emails = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
       String comment = (String) entity.getProperty("comment-string");
+      String email = (String) entity.getProperty("email");
```

Comment:
What happens if any of these values are not set?

Reply:
The email property will always be set, because the user can't access my comments page without logging into the site (that's how I set up the authentication). And at least the way I written the code, users can post an empty comment.